### PR TITLE
create new state

### PR DESCRIPTION
**Pull request by araratyan98**

### CODE REVIEW

File: src/components/shared/state/index.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react'
+
+export default function State() {
+  const [value, setValue] = useState<string>('')
+  console.log(value, 'value')
```
Comment (ervand96) [inline src/components/shared/state/index.tsx]: please delete this

File: src/components/shared/state/index.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    setValue(e.target.value)
+  }
+  const delet = () => {
+    setValue('')
+  }
+  return (
+    <div>
+      <input onChange={hendler} value={value} type="text" />
+      <h1>{value}</h1>
+      <button onClick={delet} disabled={!value}>
```
Comment (ervand96) [inline src/components/shared/state/index.tsx]: please change name

File: src/components/shared/state/index.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react'
+
+export default function State() {
+  const [value, setValue] = useState<string>('')
+  console.log(value, 'value')
+  const hendler = (e: any) => {
```
Comment (ervand96) [inline src/components/shared/state/index.tsx]: please add type

File: src/components/shared/state/index.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    setValue(e.target.value)
+  }
+  const delet = () => {
```
Comment (ervand96) [inline src/components/shared/state/index.tsx]: the same

File: src/components/shared/state/index.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    setValue('')
+  }
+  return (
+    <div>
+      <input onChange={hendler} value={value} type="text" />
```
Comment (ervand96) [inline src/components/shared/state/index.tsx]: typo

File: src/components/shared/state/index.tsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+      <h1>{value}</h1>
+      <button onClick={delet} disabled={!value}>
+        maqrel
```
Comment (ervand96) [inline src/components/shared/state/index.tsx]: please write eng

File: src/components/shared/state/index.tsx [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react'
+
+export default function State() {
```
Comment (ervand96) [inline src/components/shared/state/index.tsx]: please add return type

**Comment by araratyan98** [comment]
🦁🐽🐗🐖🐷

чт, 22 сент. 2022 г. в 12:10, ervand96 ***@***.***>:

**Pull request merged**